client/src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,4 @@
+.toast, .toast-header {
+    background-color: rgb(255, 116, 116) !important;
```

Comment:
Let use just one type of color, like as Hex format, and try to don't use `important`